### ISSUE TITLE
console batch mode doesn't work as documented

### ISSUE THREAD

**Issue by 8forty** [orientechnologies/orientdb#5800]
The documentation here [http://orientdb.com/docs/2.1/Console-Commands.html](http://orientdb.com/docs/2.1/Console-Commands.html) says: 
```java
$ $ORIENTDB_HOME/bin/console.sh commands.txt
```
...but when I try it, it treats the file name as a command:
```java
[root@bwu4 /opt/orientdb]# orientdb-community-2.0/bin/console.sh commands.txt

OrientDB console v.2.0 (build UNKNOWN@r; 2015-01-20 16:41:17+0000) www.orientechnologies.com
Type 'help' to display all the supported commands.
Installing extensions for GREMLIN language v.2.6.0

orientdb> commands.txt

!Unrecognized command: 'commands.txt'
```

**Comment by luigidellaquila** [comment]
Hi @8forty 

it works correctly if you give it an absolute path, eg.

```
[root@bwu4 /opt/orientdb]# orientdb-community-2.0/bin/console.sh /absolute/path/to/commands.txt
```

but it does not seem to work correctly for relative paths. I'm checking it now

Thanks

Luigi

**Comment by luigidellaquila** [comment]
Hi @8forty 

I just pushed a fix on branch 2.1.x, it will be released with 2.1.13

Thanks

Luigi
